index.js

```diff
@@ -17,7 +17,12 @@ express()
   .use(express.static(path.join(__dirname, 'public')))
   .set('views', path.join(__dirname, 'views'))
   .set('view engine', 'ejs')
-  .get('/', (req, res) => res.render('pages/index'))
+  .get('/', jsonParser, async (req, res) => {
+    try {
+      const client = await pool.connect();
+      const result = await client.query(`SELECT * FROM fridge_products`);
+      const results = { 'results': (result) ? result.rows : null};
+      res.render('pages/index', results );
```

Comment:
It looks like you might be missing a few lines here. The try/catch statements are opened here but aren't closed, which will likely lead to syntax errors as the code is expecting the try statement to be closed with a curly bracket and a subsequent catch statement to follow. Perhaps this accidentally got cut off somewhere when copy pasting the template from a different endpoint?

Regardless, try adding the following code just after line 26 to close the try/catch statements and resolve the syntax error:
`client.release();
    } catch (err) {
      console.error(err);
      res.send("Error " + err);
    }`

Reply:
good catch, thank you!